games-simulation/openrct2/openrct2-0.2.4.ebuild

```diff
@@ -0,0 +1,182 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils xdg-utils
+
+MY_PN="OpenRCT2"
+MY_PN_OBJ="objects"
+MY_PN_RPL="replays"
+MY_PN_TS="title-sequences"
+MY_PV_OBJ="1.0.12"
+MY_PV_RPL="0.0.4"
+MY_PV_TS="0.1.2c"
+
+DESCRIPTION="An open source re-implementation of Chris Sawyer's RollerCoaster Tycoon 2"
+HOMEPAGE="https://openrct2.org/"
+SRC_URI="
```

Comment:
While I see, that such checks can be useful in some conditions, I don't see an advantage here (for me). As I am not allowed to symlink `-9999` anyway, what's the point in writing more lines and have a more 'not so clean layout (sorry, don't know how to better say)', which is used here  for extra inherit, SRC_URI and unpacking? At the end, I still have could have two ebuilds with possible differences. Since I am going to maintain that package, I like it more that way.

Reply:
So the idea of that structure is that you keep the live ebuild up to date, and whenever there is a new release, you copy the live ebuild as the new version. This in contrast to possibly copying the latest ebuild and editing from there based on the changes in the new version. Ultimately the decision is up to the maintainer, I wanted to know your logic in it.

Reply:
I see your point. If you do it that way, you could copy it the other way round as a new stable released. I will keep this in mind for future submissions, but leave it for now, as it is.

Reply:
That's fine, we can do that unless proxy-maint objects.